capstone/src/main/java/com/google/sps/servlets/AnnouncementsServlet.java

```diff
@@ -12,18 +25,57 @@
 @WebServlet("/announcements")
 public class AnnouncementsServlet extends HttpServlet {
 
+  private static final String ANNOUNCEMENTS_CONTENT_PROP = "content";
+  private static final String ANNOUNCEMENT_PROP = "Announcement";
+  private static final String AUTHOR_PROP = "author";
+  private static final String TIME_PROP = "time";
+  private static final String CONTENT_PROP = "content";
+  private static final String CLUB_PROP = "club";
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Is this needed for the prototype? If not, it seems like you're starting to get into the main implementation since you're adding Datastore. Could you start adding tests for the servlets?